src/Workspaces/Core/Portable/Storage/DesktopPersistenceStorageServiceFactory.cs

```diff
@@ -36,12 +37,16 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
 #else
 
         private readonly SQLiteConnectionPoolService _connectionPoolService;
+        private readonly IAsynchronousOperationListener _asyncListener;
 
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-        public DesktopPersistenceStorageServiceFactory(SQLiteConnectionPoolService connectionPoolService)
+        public DesktopPersistenceStorageServiceFactory(
+            SQLiteConnectionPoolService connectionPoolService,
+            IAsynchronousOperationListenerProvider asyncOperationListenerProvider)
         {
             _connectionPoolService = connectionPoolService;
+            _asyncListener = asyncOperationListenerProvider.GetListener(FeatureAttribute.Workspace);
```

Comment:
@sharwell wasn't certain what listener to get for thsi guy.

Reply:
I'd add `FeatureAttribute.PersistentStorage` for this.